lib/ansible/modules/database/mysql/mysql_replication.py

```diff
@@ -160,6 +161,15 @@ def reset_slave(cursor):
     return reset
 
 
+def reset_master(cursor):
+    try:
+        cursor.execute("RESET MASTER")
+        reset = True
+    except:
```

Comment:
You've probably done it for consistency with the rest of the module, but this should handle specifically anticipated exceptions here. At the minimum it should be `except Exception:`. Here's an open pull request that's fixing the rest of those in this module: https://github.com/ansible/ansible/pull/45389